wp-admin/js/widgets/media-image-widget.js

```diff
@@ -69,6 +88,38 @@
 		},
 
 		/**
+		 * Get the instance props from the media selection frame.
+		 *
+		 * @param {object} attachment Attachment object.
+		 * @returns {object} Props
+		 */
+		_getEmbedProps: function _getAttachmentProps( attachment ) {
+			var props = {};
+
+			if ( ! _.isEmpty( attachment ) ) {
+				props = {
+					attachment_id: attachment.id,
+					align: attachment.align,
+					alt: attachment.alt,
+					caption: attachment.caption,
+					image_classes: '',
+					image_title: '',
+					link_classes: '',
+					link_rel: '',
+					link_url: attachment.linkUrl ? attachment.linkUrl : attachment.link,
```

Comment:
I thought link was the link type (self, attachment page, other) and the linkUrl was the actual URL. I will test this out.